### ISSUE TITLE
Ajax call no longer defaults to 'body' in some cases

### ISSUE THREAD

**Issue by RussBaz** [bigskysoftware/htmx#2945]
I have recently updated to 2.x and my code, that used to work, started failing:
```js
htmx.ajax('GET', next, {
    header: config.headers
});
```
However, if I add an explicit target, it works as it used to:
```js
htmx.ajax('GET', next, {
    target: 'body',
    header: config.headers
});
```

I have not found this documented. Is this a bug or a missing piece of documentation only?

**Comment by MichaelWest22** [comment]
I think the issue may be a recent related bug fix for the ajax Helper function to prevent it defaulting to target body when you pass in bad targets/source fixed in #2869 #2878.

I would try testing with 2.0.2 instead of 2.0.3 and see if this resolves your issue.  I Think I can fix the logic to handle it better with another PR.

**Comment by 1cg** [comment]
Thank you @MichaelWest22 I will cut another release in November w/ the fix
